v2/brigadier/src/events.ts

```diff
@@ -1,26 +1,58 @@
 import { Worker } from "./workers"
 import { Project } from "./projects"
 
+/**
+ * An event received by Brigade that resulted in triggering an event handler.
+ */
 export interface Event {
+  /** A unique identifier for the event. */
   id: string
+  /** The project that registered the handler being called for the event. */
   project: Project
+  /** The gateway which created the event. */
```

Comment:
It's worth it to mention here that it's a unique identifier.

Docs on creating or configuring gateways will emphasize that the value of the `source` field on any event, in order to avoid collisions, should be a URL that is controlled exclusively by the author(s) of the gateway. So, for instance, the GitHub gateway I am working on right now uses a `source` value `github.com/brigadecore/brigade-github-gateway` (although I'm realizing now that `brigade.sh/github-gateway` would be more succinct and fulfill the same purpose). This is pretty much the same as Java's package naming strategy of using your own domain name in reverse as a package name prefix to avoid package name collisions (e.g. `com.example`).

Not all of this info needs to be captured here, but it's worth adding the word "unique" here.